packages/iota/test/validation/transactionEssence.spec.ts

```diff
@@ -0,0 +1,147 @@
+// Copyright 2020 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+import { ED25519_ADDRESS_TYPE } from "../../src/models/addresses/IEd25519Address";
+import { UTXO_INPUT_TYPE } from "../../src/models/inputs/IUTXOInput";
+import { BASIC_OUTPUT_TYPE } from "../../src/models/outputs/IBasicOutput";
+import { TREASURY_OUTPUT_TYPE } from "../../src/models/outputs/ITreasuryOutput";
+import { ADDRESS_UNLOCK_CONDITION_TYPE } from "../../src/models/unlockConditions/IAddressUnlockCondition";
+import { validateInputs } from "../../src/validation/inputs/inputs";
+import { validateOutputs } from "../../src/validation/outputs/outputs";
+import { cloneTransactionEssence } from "./testUtils";
+import { mockTransactionEssence, protocolInfoMock, mockMaxDistintNativeTokens } from "./testValidationMocks";
+
+describe("Parents validation", () => {
+    it("should fail with inputs count must be greater then equal to 1 and less then equal to 128", () => {
```

Comment:
"Greater than equal to 1" also known as "greater than 0".
But just say `should fail when input count is out of bounds` so my brain doesn't explode reading the sentence.